sound/soc/sof/ipc4-topology.c

```diff
@@ -1318,6 +1403,37 @@ static int sof_ipc4_prepare_mixer_module(struct snd_sof_widget *swidget,
 	return sof_ipc4_widget_assign_instance_id(sdev, swidget);
 }
 
+static int sof_ipc4_prepare_process_module(struct snd_sof_widget *swidget,
+					 struct snd_pcm_hw_params *fe_params,
+					 struct snd_sof_platform_stream_params *platform_params,
+					 struct snd_pcm_hw_params *pipeline_params, int dir)
+{
+	struct snd_soc_component *scomp = swidget->scomp;
+	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(scomp);
+	struct sof_ipc4_process *process = swidget->private;
+	int ret;
+
+	process->available_fmt.ref_audio_fmt = &process->available_fmt.base_config->audio_fmt;
+
+	/* output format is not required to be sent to the FW for kpb */
```

Comment:
KPB? there's no mention of KPB yet anywhere else? is this true for all process components?

Reply:
@ujfalusi I haven't summited the new PR. I will close this PR and split it into several PRs. : )